components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/util/SSOConstants.java

```diff
@@ -57,7 +57,7 @@ public class SSOConstants {
     public static final String SAML_SLO_URL = "/identity/saml/slo";
     public static final Pattern SAML_SLO_ENDPOINT_URL_PATTERN = Pattern.compile("(.*)/identity/saml/slo/?");
 
-    public static final String AUTHN_REQUEST_PROVIDER_NAME = "AuthnReqProviderName";
```

Comment:
Do we need to remove this? Since it's a public constant, unless we are certain that there are no other use cases, better to keep it.

Reply:
This constant was added with my recent PR [1], and is therefore only used here.

[1] https://github.com/wso2-extensions/identity-outbound-auth-samlsso/pull/182